dsmr_parser/obis_name_mapping.py

```diff
@@ -93,30 +93,30 @@
     'Q3D_EQUIPMENT_STATE': 'Q3D_EQUIPMENT_STATE',
     'Q3D_EQUIPMENT_SERIALNUMBER': 'Q3D_EQUIPMENT_SERIALNUMBER',
     'BELGIUM_MBUS2_DEVICE_TYPE': 'BELGIUM_MBUS2_DEVICE_TYPE',
-	'EON_HU_COSEM_LOGICAL_DEVICE_NAME': 'COSEM_LOGICAL_DEVICE_NAME',
-	'EON_HU_EQUIPMENT_SERIAL_NUMBER': 'EQUIPMENT_SERIAL_NUMBER',
-    'EON_HU_ELECTRICITY_DELIVERED_TARIFF_3': 'ELECTRICITY_DELIVERED_TARIFF_3',
-    'EON_HU_ELECTRICITY_DELIVERED_TARIFF_4': 'ELECTRICITY_DELIVERED_TARIFF_4',
-    'EON_HU_ELECTRICITY_USED_TARIFF_3': 'ELECTRICITY_USED_TARIFF_3',
-    'EON_HU_ELECTRICITY_USED_TARIFF_4': 'ELECTRICITY_USED_TARIFF_4',
-    'EON_HU_ELECTRICITY_REACTIVE_TOTAL_Q1': 'ELECTRICITY_REACTIVE_TOTAL_Q1',
-    'EON_HU_ELECTRICITY_REACTIVE_TOTAL_Q2': 'ELECTRICITY_REACTIVE_TOTAL_Q2',
-    'EON_HU_ELECTRICITY_REACTIVE_TOTAL_Q3': 'ELECTRICITY_REACTIVE_TOTAL_Q3',
-    'EON_HU_ELECTRICITY_REACTIVE_TOTAL_Q4': 'ELECTRICITY_REACTIVE_TOTAL_Q4',
-    'EON_HU_ELECTRICITY_COMBINED': 'ELECTRICITY_COMBINED',
-    'EON_HU_INSTANTANEOUS_POWER_FACTOR_TOTAL': 'INSTANTANEOUS_POWER_FACTOR_TOTAL',
-    'EON_HU_INSTANTANEOUS_POWER_FACTOR_L1': 'INSTANTANEOUS_POWER_FACTOR_L1',
-    'EON_HU_INSTANTANEOUS_POWER_FACTOR_L2': 'INSTANTANEOUS_POWER_FACTOR_L2',
-    'EON_HU_INSTANTANEOUS_POWER_FACTOR_L3': 'INSTANTANEOUS_POWER_FACTOR_L3',
-    'EON_HU_FREQUENCY': 'FREQUENCY',
-    'EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q1': 'INSTANTANEOUS_REACTIVE_POWER_Q1',
-    'EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q2': 'INSTANTANEOUS_REACTIVE_POWER_Q2',
-    'EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q3': 'INSTANTANEOUS_REACTIVE_POWER_Q3',
-    'EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q4': 'INSTANTANEOUS_REACTIVE_POWER_Q4',
-    'EON_HU_MAX_POWER_ON_L1': 'MAX_POWER_ON_L1',
-    'EON_HU_MAX_POWER_ON_L2': 'MAX_POWER_ON_L2',
-    'EON_HU_MAX_POWER_ON_L3': 'MAX_POWER_ON_L3',
-    'EON_HU_LAST_MONTH_DATA': 'LAST_MONTH_DATA'
+	'EON_HU_COSEM_LOGICAL_DEVICE_NAME': 'EON_HU_COSEM_LOGICAL_DEVICE_NAME',
```

Comment:
Should these names really have the EON_HU prefix? Can't we use general names for nearly all values?

Reply:
Yes, this is the main problem with this. Also a lot of 3rd party app uses the reference ID-s, so we can't change them for backward compatibility.
Maybe it is safe to change the value_name (in `obis_name_mapping.py`) and have some clarification there of the multiple (almost) identical names. But I'm not sure if this will cause breakign change.
But I defienietly would not change the OBIS reference codes (in `obis_references.py`).